models/player_encrypted.go

```diff
@@ -0,0 +1,13 @@
+// khan
+// https://github.com/topfreegames/khan
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2016 Top Free Games <backend@tfgco.com>
+
+package models
+
+// PlayerEncrypted identifies uniquely one player in a given game
+type PlayerEncrypted struct {
```

Comment:
Could we rename it to `EncryptedPlayer` to follow the English grammatical rules?

Reply:
![Open-English-Logo_og](https://user-images.githubusercontent.com/1509421/112665315-b1ab3680-8e39-11eb-8c2c-5cb70f783757.jpeg)